docs/source/authentication.rst

```diff
@@ -4,29 +4,62 @@
+Types of authentication
+========================
+You can authenticate with |company| on a user basis by registering an :xref:`ssh_public_key_authentication` (i.e. "public key" in this document), or on a specific project or dataset basis by using an |company|-generated API key tied to that resource.
+
+User authentication via public key (**recommended**)
+------------------------------------------------------------------------
+Public key authentication gives you full access to all the capabilities of the SDK.
+**This is our recommended default authentication method.**
+
+Public key authentication is required to use the :class:`.EncordUserClient`.
 
-* :ref:`authentication:Register a public key`: registering a public key gives you full access, allowing you to access all your projects and datasets, as well as create new projects, datasets and API keys
-* :ref:`authentication:Create an API key`: creating an API key gives you restricted access to a particular project or dataset
+You can use the client to interact with a :ref:`general_concepts:Dataset` or a :ref:`general_concepts:Project` assuming you are either of the following:
 
-Public keys are used to authenticate the user for activities that require user-level access, for example creating projects and datasets.
-API keys are more restrictive and are tied to a particular project or dataset.
-API keys are used to authenticate calls that only require resource-level access, for example downloading label rows from a particular project.
+* An organisation admin of the organisation this project or dataset is part of
+* A project admin or project manager of the project that is being accessed
+* A dataset admin of the dataset that is being accessed
 
+Resource authentication via API key (supports legacy flows)
+------------------------------------------------------------------
+You can access a specific project or dataset via API key authentication.
+An individual API key is always tied to one specific project or dataset.
+**This is an authentication flow that will be deprecated in the future in favour of the public key authentication.**
+
+We only recommend you to use this authentication if
+
+* You do not have user access rights for a project or dataset
+* You want to access a project or dataset as part of an automated pipeline without using a specific user account
+
+User authentication
+==========================
 
-Register a public key
----------------------
+Set up authentication keys - user authentication
+--------------------------------------------------------------------
 Public keys are tied to the user so need to be added to your profile.
 To register a public key, please follow the instructions in :xref:`public_keys` of the |sdk| documentation.
 
-Create an API key
------------------
 
+Authenticate with |company| - user authentication
```

Comment:
```suggestion
Authenticate with |company|
```

Reply:
I cannot do this because headings need to be unique strings in sphinx.